screeps-start.js

```diff
@@ -125,8 +126,8 @@ const writeModsConfiguration = () => {
 
 // Map from camelCase to snake_case
 const ServerConfigMap = {
-  runnersThreads: "runner_threads",
-  processorsCnt: "processors_cnt",
+  runnerCount: "runners_threads",
```

Comment:
runners_threads is not a great name to determine the runners **count**. We should get rid of runner_threads completely

Reply:
Done, was a miss on my end. It will be runner count and threads are gone.